src/components/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default function Navbar() {
+  const navStyle = {
+    display: 'flex',
+    justifyContent: 'center',
+  };
+  const ulStyle = {
+    display: 'flex',
+    gap: '1rem',
+    listStyle: 'none',
+  };
+  return (
+    <nav style={navStyle}>
+      <ul style={ulStyle}>
+        <li><NavLink to="/" /></li>
```

Comment:
- [ ] It would be better if you add the link to move to the books page. kindly fix this minor issue.

```suggestion
        <li><NavLink to="/">Books </NavLink></li>
```